src/actions/auth/loginAction.js

```diff
@@ -0,0 +1,35 @@
+import axios from 'axios';
+import { toast } from 'react-toastify';
+import 'react-toastify/dist/ReactToastify.css';
+import { LoginConstants } from '../actionTypes';
```

Comment:
@Manorlds-Eaglespark, No need for importing `import 'react-toastify/dist/ReactToastify.css';` because it is already imported in the App.jsx file.

Reply:
Awesome. done.